Document/0x05b-Basic-Security_Testing.md

```diff
@@ -1256,15 +1314,7 @@ package:/data/app/com.awesomeproject-1/base.apk
 $ adb pull /data/app/com.awesomeproject-1/base.apk
 ```
 
-`apkx` provides an easy method of retrieving an APK's source code via the command line. It also packages `dex2jar` and CFR and automates the extraction, conversion, and decompilation steps. Install it as follows:
-
-```shell
-$ git clone https://github.com/b-mueller/apkx
-$ cd apkx
-$ sudo ./install.sh
-```
-
-This should copy `apkx` to `/usr/local/bin`. Run it on the APK that you want to test as follows:
+`apkx` provides an easy method of retrieving an APK's source code via the command line. It also packages `dex2jar` and CFR and automates the extraction, conversion, and decompilation steps. See [Apkx installation instructions](#####Apkx "Apkx installation instructions"). Once installed, run it against a local APK as follows:
```

Comment:
Also this link. Please check if there are more of these broken links. Thanks!

Reply:
Fixed all broken links.